docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -240,6 +234,19 @@
         max-width: 100%;
       }
     }
+
+    &--has-date {
+      // This ensures there is always enough height for the date tile in case
+      // the images is too short (can happen with util classes)
+      min-height: hb-calculate-rems(130px);
+    }
+
+    .hb-card--no-image &,
```

Comment:
can you add a little comment here about what this is doing to allow the text to flow over to the left if there is no image?